http-gateway/web/src/containers/things/things-list-page.js

```diff
@@ -26,7 +26,7 @@ export const Things = () => {
         Header: _(t.name),
         accessor: 'device.n',
         Cell: ({ value, row }) => (
-          <Link to={`/things/${row.values.id}`}>{value}</Link>
+          <Link to={`/things/${row.values?.['device.di']}`}>{value}</Link>
```

Comment:
is "di" correct?  If it is, I think it should be deviceIndentfier or something else like that.   id is clear, di is not

Reply:
That's how it comes from the API (`di` stands for device ID), I can't do much about it :)